From 053fb0092e4a4fd005f43224ddb8b797121ef151 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Mon, 25 Jun 2018 14:56:49 +0900
Subject: [PATCH 08/25] tmpfiles: specify access mode for /run/systemd/netif

This partially reverts 2af767729489f6baa98a2641b2007acab44ed353.
As the directories are certainly readable and not-writable by
non-privileged users.

(cherry picked from commit 400d8461381c5748a97fccb8a022a20f40063a7d)
---
 tmpfiles.d/systemd.conf.m4 | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tmpfiles.d/systemd.conf.m4 b/tmpfiles.d/systemd.conf.m4
index 0992f74209..120e42a1b9 100644
--- a/tmpfiles.d/systemd.conf.m4
+++ b/tmpfiles.d/systemd.conf.m4
@@ -17,9 +17,9 @@ d /run/systemd/users 0755 root root -
 d /run/systemd/machines 0755 root root -
 d /run/systemd/shutdown 0755 root root -
 m4_ifdef(`ENABLE_NETWORKD',
-d /run/systemd/netif - - - -
-d /run/systemd/netif/links - - - -
-d /run/systemd/netif/leases - - - -
+d /run/systemd/netif 0755 - - -
+d /run/systemd/netif/links 0755 - - -
+d /run/systemd/netif/leases 0755 - - -
 )m4_dnl
 
 d /run/log 0755 root root -
-- 
2.19.1

